runner/src/server/plugins/engine/pageControllers/RepeatingSummaryPageController.ts

```diff
@@ -170,14 +170,21 @@ export class RepeatingSummaryPageController extends PageController {
   buildTextFieldRows(answers, form_session_identifier, view = false) {
     const { title = "" } = this.inputComponent;
     return answers?.map((value, i) => {
-      const valueValues: string[] = [];
+      const rowValues: string[] = [];
       for (const key in value) {
-        if (this.inputComponent.getComponentType(key) == "DatePartsField") {
-          valueValues.push(format(parseISO(value[key]), "d/MM/yyyy"));
-        } else {
-          valueValues.push(
-            `${this.inputComponent.getPrefix(key)}${value[key]}`
+        const componentType = this.inputComponent.getComponentType(key);
+        //TODO would a switch statement be better here
```

Comment:
imo no, not a fan of switch statements, maybe a map of component type to formatting function would be good though, but given there are only 2, think its fine as an if right now

Reply:
Yeah I put it as a todo to keep it open for discussion when we have to add more component types in